### DISCUSSION TITLE
Optimistic Concurrency Control Use UpdateMany Has Problems

### DISCUSSION THREAD

**Discussion by AoiIsMine** [prisma/prisma#10254] [Q&A]
I have a problem. I don't know if I use it wrong or there is a problem
Under concurrency, optimistic locks implemented with updateMany will have overwrite writes
Thank u for your help~
[issues](https://github.com/prisma/prisma/issues/10207)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
